Cakes-Co/index.html

```diff
@@ -7,10 +7,108 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
+     <link rel="stylesheet" href="style.css">
+     <link href="https://fonts.googleapis.com/css2?family=Pacifico&family=Poppins:wght@400;700&display=swap" rel="stylesheet">
 </head>
-
 <body>
     <!-- Add your markup here -->
+
+
+
+
+   
+<header class="header">
```

Comment:
Perhaps, there's no need to use `class="header"` here and apply style on `header` directly. Let me know what you think!